cap/modules/records/serializers/schemas/json.py

```diff
@@ -50,6 +56,25 @@ def can_user_admin(self, obj):
         deposit = CAPDeposit.get_record(obj['pid'].object_uuid)
         return AdminRecordPermission(deposit).can()
 
+class RecordFormSchema(RecordSchema):
+    """Schema for records v1 in JSON."""
+
+    schemas = fields.Method('get_record_schemas', dump_only=True)
+
+    def get_record_schemas(self, obj):
+        deposit = CAPDeposit.get_record(obj['pid'].object_uuid)
+
+        schema = current_jsonschemas.get_schema(
+                    deposit.schema.record_path,
+                    with_refs=True,
+                    resolved=False
+                )
+        uiSchema = deposit.schema.deposit_options
```

Comment:
this will work for now (as in all the cases we keep only deposit_options), but after #1260 this should refer to schema.record_options (and if flag use_deposit_as_record is on, deposit one will be served)